doc/ipythonconsole.rst

```diff
@@ -41,10 +41,15 @@ To connect to an external kernel,
 #. Launch an IPython kernel on the local or remote host if one is not already running.
    If using Spyder 3.3.0 or later, you'll need to do so with ``python -m spyder_kernels.console`` (after you've first installed ``spyder-kernels`` on the host with ``<conda/pip> install spyder-kernels``).
    If using a version of Spyder before 3.3.0, ``ipython kernel`` should work to launch the kernel, albeit without certain Spyder-specific features.
-#. Make sure to copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
+#. Copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
+   You can get the location of :file:`{runtime-dir}` by running the command ``import jupyter_core; jupyter_core.paths.jupyter_runtime_dir()`` in an IPython interpreter launched in the same Python environment as the kernel you launched.
```

Comment:
This is good, but *please* add the actual code block that people need to run, as I suggested before.

From all my years giving support, I'm certain people will understand more easily what to do if they see the code in an IPython session, instead of reading the description in words.

Reply:
I'm not sure I understand what you are asking, sorry. ``import jupyter_core; jupyter_core.paths.jupyter_runtime_dir()`` *is* the actual code block they need to run, identical to the example you gave, in the same ``code`` notation, just not block style.

If you're talking about the latter, they still need to read the prose text to understand what the output means and how to use it; blindly copying and pasting it and running doesn't do anything by itself so nothing is gained by encouraging them to run it even if they don't bother to read the text (unlike stuff like ``conda update spyder``, ``spyder --reset``, etc). Conversely, embedding it in the prose makes it more likely people will read said text, and in any case it seems inappropriate to allocate a whole code block to a single-line command that itself part of one sidenote to a list item.

Reply:
I don't want to enter an endless discussion here. Please do the change as I suggested in my initial comment or I'll do it.

Reply:
...

Done as demanded.